### PR TITLE
Spellings

### PR DESCRIPTION
**Pull request by haraldgroven**
Minor corrections to en.json discovered while translating it to another language. 

- Swaziland changed name in 2018
- In company names: Lowercase articles (a, an, the), coordinating conjunctions, and prepositions (regardless of length).

### CODE REVIEW

File: web/.husky/pre-commit
Comment (VIKTORVAV99) [inline web/.husky/pre-commit:1]: Why did you remove this?

Comment (haraldgroven) [reply web/.husky/pre-commit:1]: I had to remove the directory locally because [it was causing git errors](https://www.google.com/search?q=github+web/.husky/pre-commit:+line+5:+pnpm:+command+not+found+husky+-+pre-commit+hook+exited+with+code+127+(error)+site:stackoverflow.com). 
It was my mistake not to see that I had accidentally committed the change to the language file.

Comment (VIKTORVAV99) [reply web/.husky/pre-commit:1]: It caused git errors because it was unable to run the pre-commit hooks properly. This is by design and is intended to ensure that changes passes a minimum validation before getting pushed.

**Review by VIKTORVAV99 — APPROVED** [review]
LGTM!

Thanks for updating these!
